meta-client/src/main/java/org/apache/helix/metaclient/factories/MetaClientConfig.java

```diff
@@ -0,0 +1,91 @@
+package org.apache.helix.metaclient.factories;
+
+class MetaClientConfig {
+
+  public enum RetryProtocol {
+    NO_RETRY, EXP_BACK_OFF, CONST_RETRY_INTERVAL
+  }
```

Comment:
RetryProtocol should better be a standalone class, we may want to add their custom logic and make it self-contained.

Reply:
I removed the `RetryProtocol` enum. We could add back when we introduce retry config.